runelite-client/src/main/java/net/runelite/client/config/Keybind.java

```diff
@@ -39,15 +40,15 @@
 @EqualsAndHashCode
 public class Keybind
 {
-	private static final BiMap<Integer, Integer> modifierToKeyCode = new ImmutableBiMap.Builder<Integer, Integer>()
+	private static final BiMap<Integer, Integer> MODIFIER_TO_KEY_CODE = new ImmutableBiMap.Builder<Integer, Integer>()
```

Comment:
.inverse is not very cheap afaik, you should probably declare static MODIFIER_TO_KEY_CODE_INVERSED or something

Reply:
It is cached internally

Reply:
I see, alright